docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,235 @@
+=========================================
+PostgreSQL specific aggregation functions
+=========================================
+
+.. module:: django.contrib.postgres.aggregates
+   :synopsis: PosgreSQL specific aggregation functions
+
+All of these functions are available
+from the ``django.contrib.postgres.aggregates``.
+
+.. warning::
+
+    All functions come without default aliases,
+    so you must explicitly provide one.
+    Usage example::
+
+        >> result = SomeModel.objects.aggregate(arr=ArrayAgg('somefield'))
+        >> print(result)
+        {'arr': [0, 1, 2]}
+
+
+General-purpose aggregation functions
+-------------------------------------
+
+ArrayAgg
+~~~~~~~~
+
+.. class:: ArrayAgg(expression, **extra)
+
+    Returns input values, including nulls, concatenated into an array.
+
+    * Return type: ``list``
+
+BitAnd
+~~~~~~
+
+.. class:: BitAnd(expression, **extra)
+
+    Returns the bitwise ``AND`` of all non-null input values, or ``NULL`` if none.
+
+    * Return type: ``int`` or ``None``
+
+BitOr
+~~~~~
+
+.. class:: BitOr(expression, **extra)
+
+    Returns the bitwise ``OR`` of all non-null input values, or ``NULL`` if none.
+
+    * Return type: ``int`` or ``None``
+
+BoolAnd
+~~~~~~~~
+
+.. class:: BoolAnd(expression, **extra)
+
+    Returns ``True``, if all input values are true, otherwise ``False``.
+
+    * Return type: ``bool`` or ``None``
+
+BoolOr
+~~~~~~
+
+.. class:: BoolOr(expression, **extra)
+
+    Returns ``True`` if at least one input value is true, otherwise ``False``.
+
+    * Return type: ``bool`` or ``None``
+
+
+StringAgg
+~~~~~~~~~
+
+.. class:: StringAgg(expression, delimiter)
+
+    Returns input values concatenated into a string, separated by delimiter.
+
+    * Return type: ``str`` or ``None``
+
+    .. attribute:: delimiter
+
+        Required argument. Needs to be a string. Examples: ``:``, ``;``.
```

Comment:
Not sure this is really needed given the above explanation. Shouldn't need to give examples of strings.